NDB.Covid19/NDB.Covid19/WebServices/ImportantMessageService.cs

```diff
@@ -13,6 +15,8 @@ public async Task<ImportantMessage> GetImportantMessage()
             ApiResponse<ImportantMessage> response = await Get<ImportantMessage>($"{Conf.URL_GET_IMPORTANT_MESSAGE}?lang={LocalPreferencesHelper.GetAppLanguage()}");
             HandleErrorsSilently(response);
 
+            ServiceLocator.Current.GetInstance<IDeveloperToolsService>().SaveLastFetchedImportantMessage(response, DateTime.UtcNow) ;
```

Comment:
There is a special class that is used for getting current UTC time, see in other places in thee code. Use it instead.